spec/controllers/admin/admin_users_controller_spec.rb

```diff
@@ -92,16 +92,63 @@
         it_redirects_to_with_error(root_url, "Sorry, only an authorized admin can access the page you were trying to reach.")
       end
     end
-  
+
     context "when admin has correct authorization" do
       role = FactoryBot.create(:role)
+      before { fake_login_admin(admin) }
 
-      it "updates user roles" do
-        admin.update(roles: ["policy_and_abuse"])
-        fake_login_admin(admin)
-        put :update, params: { id: user.login, user: { roles: [role.id.to_s] } }
+      context "when admin has superadmin role" do
+        before { admin.update(roles: ["superadmin"]) }
+
+        it "allows admins to update all attributes" do
+          put :update, params: {
+            id: user.login,
+            user: {
+              email: "updated@example.com",
+              roles: [role.id.to_s]
+            }
+          }
+
+          it_redirects_to_with_notice(root_path, "User was successfully updated.")
+        end
+      end
+
+      %w(open_doors tag_wrangling).each do |admin_role|
```

Comment:
I'm having second thoughts about this style... I think because it's an array, it looks better with `[]`.

Reply:
I agree -- the brackets are much more intuitive for an array.

Reply:
I'll update .rubocop.yml and clean up the `%w` brackets in #3868 (at least the ones that got added this release).